src/server/db_setup.ts

```diff
@@ -33,6 +37,40 @@ async function init(): Promise<void> {
         // release the client back to the pool when we're done
         await client.release();
     }
+
+    //for importing game logs from csv file
+    var stream = fs.createReadStream("/app/db-migrations/GameLogs.csv");
```

Comment:
Per CircleCI test results: https://app.circleci.com/pipelines/github/gik0geck0/DominionStats/49/workflows/7b61b5a4-9682-49b9-abb5-00f9a66bd247/jobs/90

Use const or let here (and basically never use var anymore)